regen-network/regen-web
# fix: APP-83 keep unapproved projects on public profile projects tab

## Description

https://regennetwork.atlassian.net/browse/APP-83?atlOrigin=eyJpIjoiMmIxMTY3ZTUzYTYzNGE3ZTkwNmNiOTQ4ZWQ1ZTA5ODAiLCJwIjoiaiJ9



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



From https://deploy-preview-2334--regen-marketplace.netlify.app/
1. log in and create a project (or edit an existing unpublished project) and publish it.
2. go to your **public** profile page to check the project is there, although it's not approved yet. The unpublished projects are still not there. On your private profile page, you can still see both unpublished and unapproved projects.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
